client/my-sites/site-settings/disconnect-site/troubleshoot.jsx

```diff
@@ -9,12 +9,21 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import JetpackConnectHappychatButton from 'jetpack-connect/happychat-button';
 import HelpButton from 'jetpack-connect/help-button';
+import JetpackConnectHappychatButton from 'jetpack-connect/happychat-button';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
 import { recordTracksEvent, withAnalytics } from 'state/analytics/actions';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
 
-const Troubleshoot = ( { trackSupportClick, translate } ) => (
-	<div className="disconnect-site__troubleshooting">
+const Troubleshoot = ( { siteSlug, trackDebugClick, trackSupportClick, translate } ) => (
+	<LoggedOutFormLinks>
+		<LoggedOutFormLinkItem
+			href={ 'https://jetpack.com/support/debug/?url=' + siteSlug }
```

Comment:
I don't think this will work well for non-root sites. For example, a site at `https://example.com/not-root` will have the slug `example.com::not-root` which fails on the debug page.

Something like `addQueryArgs(  { url: siteUrl }, 'https://jetpack.com/support/debug/' )` would probably be best.

Reply:
10f7382